packages/ruby.rb

```diff
@@ -3,44 +3,41 @@
 class Ruby < Package
   description 'Ruby is a dynamic, open source programming language with a focus on simplicity and productivity.'
   homepage 'https://www.ruby-lang.org/en/'
-  version '3.3.0' # Do not use @_ver here, it will break the installer.
+  version '3.3.0-2f65458' # Do not use @_ver here, it will break the installer.
```

Comment:
Does it make sense to wait until the next stable release?  I only say this because the maintainers probably have a good reason for not tagging a new release.

Reply:
This is for a security fix, so it might make sense to get this through before a new stable.

Reply:
Also, this is from their stable branch... the update will trigger a gem rebuild for any vulnerable versions installed.